### ISSUE TITLE
rowStyle formatter no longer working in releases > 1.23.2

### ISSUE THREAD

**Issue by designdisorder** [wenzhixin/bootstrap-table#7582]
### Bootstraptable version(s) affected

1.23.3, 1.23.5

### Description

Nothing to add here

### Example(s)

https://examples.bootstrap-table.com/#options/row-style.html

**Comment by wenzhixin** [comment]
During the refactoring process, we discovered that previously returning the `rowStyle` for `css` and `classes` to `td` was incorrect; it should only take effect in `tr`. 

We can see that the HTML elements are practical, but we still need to update our example. 
<img width="369" alt="image" src="https://github.com/user-attachments/assets/008920b8-3d44-44ef-87d2-a6c96a8409ea">

In the example, we set the `background-color` and `color` for `tr`, but since `td` also has these two properties set by default, the example did not work. 

After I made the necessary modifications, it is now functioning correctly.

The working example: https://live.bootstrap-table.com/code/wenzhixin/18306
